src/index_runner/es_indexer.py

```diff
@@ -174,6 +174,8 @@ def _write_to_elastic(data):
         index - index name
         delete - bool (for delete events)
     """
+    if not data:
```

Comment:
Sometimes `data` is empty, and we want to avoid making any request to Elasticsearch in this case.